xcode/Subconscious/Shared/Components/Deck/DeckTheme.swift

```diff
@@ -15,11 +15,12 @@ extension DeckTheme {
         blendDuration: 0
     )
     
-    static let dragTargetSize = CGSize(width: 8, height: 400)
+    static let dragTargetSize = CGSize(width: 16, height: 400)
```

Comment:
Swipe confirmation bars are more obvious at this width, could be wider again?

![simulator_screenshot_6A7BB07A-7322-4EF4-8469-F02DC422A800](https://github.com/subconsciousnetwork/subconscious/assets/5009316/4fb7bea3-ce0f-467d-bc58-97fe9ec4c72b)